db/sqitch.plan

```diff
@@ -23,3 +25,4 @@ tables/application_status 2022-07-08T13:27:08Z Marcel Mueller <marcel@button.is>
 tables/attachment [tables/application] 2022-07-07T23:19:04Z Marcel Mueller <marcel@button.is> # add table attachment
 functions/open_intake 2022-08-16T13:48:54Z Anthony Bushara <anthony@button.is> # Function to display the current open intake
 mutations/submit_application 2022-08-23T17:18:10Z Matthieu Foucault <matthieu@button.is> # create a mutation to submit an application
+
```

Comment:
The changes to this file should be undone as we have a separate sqitch plan for it

Reply:
Moved to separate sqitch plan